lib/api/database/user_data.dart

```diff
@@ -43,7 +43,15 @@ class UserData {
     try {
       User res = await account.get();
       File? result = await storage.createFile(
-        file: await MultipartFile.fromPath('file', filePath, filename: imgName),
+        bucketId: 'default',
+        file: InputFile(
```

Comment:
Now it takes a bucketId and the file parameter expects `InputFile which expects MultiPartFile.

Learn more about it : https://pub.dev/documentation/appwrite/latest/appwrite/InputFile-class.html